lib/realtime/tenants/repo/migrations/20241130184212_recreate_entity_index_using_btree.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Realtime.Tenants.Migrations.RecreateEntityIndexUsingBtree do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def change do
+    execute("drop index if exists \"realtime\".\"ix_realtime_subscription_entity\"")
+
+    execute("""
+    do $$
+    begin
+      create index concurrently if not exists ix_realtime_subscription_entity on realtime.subscription using btree (entity);
+    exception
+      when others then
+        create index if not exists ix_realtime_subscription_entity on realtime.subscription using btree (entity);
+    end$$;
+    """)
```

Comment:
@filipecabaco Oli checked this and confirmed it was working.

Reply:
then merge away😃 and if we open a PR it will deploy staging 👀